### ISSUE TITLE
Crossfade adjacent selected items (move later items)

### ISSUE THREAD

**Issue by cjunekim** [reaper-oss/sws#902]
Hello

Does it actually move the later items? I have experimented that action with various timing and I could not find any movement in the items.

In what situations does the action move items?

**Comment by veto-gh** [comment]
I actually can't see anything which indicates any moving of later items too:
https://github.com/reaper-oss/sws/blob/master/Misc/ItemParams.cpp#L412

The author was probably referring to the code, which basically increases the length of the second item of any adjacent item pair to `item length + default fade lenght/2` and **moves** it to `item start - default fade lenght/2`, which (concerning the item position) is the same as dragging the left item edge to the left.

Maybe the action should be reworded "SWS: Crossfade adjacent selected items (move edges of adjacent items)" ?

**Comment by nofishonfriday** [comment]
Agreed, renaming would make sense here but I'm a little hesitant with renaming SWS actions for not breaking existing custom actions.

Not sure what's the better alternative here.
Maybe wait for Tim's thoughts ?

**Comment by wolqws** [comment]
IDs uniquely identify actions and should not change, changing their name is ok.

**Comment by veto-gh** [comment]
Thanks nofish and wol. Yes the action is referenced by `_SWS_CROSSFADE`. A concern would be users trying to find the action by filtering its name but that is moot because imo nobody will filter by the "move later items" part.

**Comment by nofishonfriday** [comment]
Since nobody seems to be against it, I've renamed the action.
Hope that's ok.

**Comment by veto-gh** [comment]
Thanks Nofish!
